app/views/search_results/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="l-2col-main">
```

Comment:
After our discussion yesterday, I think this file is redundant and unused? if that's correct can you remove this from the PR